scripts/Facilities.js

```diff
@@ -0,0 +1,19 @@
+import {getFacilities, getMinerals} from "./database.js"
+
+const facilities = getFacilities()
+const minerals = getMinerals
+
+export const facilityButtonHTMLDisplayer = () => {
```

Comment:
Maybe call this variable "FacilityButton"   ... in this case it is okay to start with a capital F